src/snowflake/snowpark/_internal/server_connection.py

```diff
@@ -626,6 +632,16 @@ def run_batch_insert(self, query: str, rows: List[Row], **kwargs) -> None:
             )
         logger.debug("Execute batch insertion query %s", query)
 
+    def _get_client_side_session_parameter(self, name: str, default_value: Any) -> Any:
```

Comment:
We also have a similar method in `session.py`. We probably should either remove the one in `session.py` to unify the usage, or make the `session.py` method to call the `server_connection.py` method. What do you think?

Reply:
Since we have access to `_conn` from session, I'd go with removing the one in `session.py`